open-api/rest-catalog-open-api.yaml

```diff
@@ -3789,6 +3986,21 @@ components:
             EmptyResponse:
               $ref: '#/components/examples/ListNamespacesEmptyExample'
 
+    PayloadTooLargeResponse:
+      description:
+        Payload is too large.
```

Comment:
Can you add here that this is how the plan route informs you that you should use the preplan route?

Because this is specific to the plan route, I think we should also make sure that this response is named to indicate that. No other routes should use it.